staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -48,6 +48,8 @@ import (
 	"k8s.io/klog"
 )
 
+const StatusHeader = "X-Updated-Status"
```

Comment:
Some ideas:
* X-Intent-Actuation
* X-Object-Change-Persisted

This is a hard one to name. I'm very certain "status" on its own won't work, and I might even go so far as to say we need to completely remove the word "status" from every name. No one is going to be able to figure this out from that name. :/

Reply:
> I might even go so far as to say we need to completely remove the word "status" from every name.

I was thinking the same

> X-Object-Change-Persisted

I like this one better.

Reply:
Same here. But still from reading the header a user could wonder why their "change" didn't get persisted (not assuming there was no change)